File: RepoTransBench/target_projects/C/Python/unix-thrust_beurk/tests/original/test_is_procnet.py
import pytest

# Simulated logic from src/is_procnet.c
def is_procnet(path):
    if not path:
        return 0
    if "/proc/net/" in path:
        return 1
    return 0

def test_procnet_tcp():
    assert is_procnet("/proc/net/tcp") == 1

def test_procnet_udp():
    assert is_procnet("/proc/net/udp") == 1

def test_not_procnet():
    assert is_procnet("/etc/passwd") == 0

def test_null_arg():
    assert is_procnet(None) == 0

File: RepoTransBench/target_projects/C/Rust/unix-thrust_beurk/tests/original/test_is_procnet.rs
// Translation of tests/unit/test_is_procnet.c

fn is_procnet(path: Option<&str>) -> i32 {
    match path {
        None => 0,
        Some(s) => {
            if s.contains("/proc/net/") { 1 } else { 0 }
        }
    }
}

#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn procnet_tcp() {
        assert_eq!(is_procnet(Some("/proc/net/tcp")), 1);
    }

    #[test]
    fn procnet_udp() {
        assert_eq!(is_procnet(Some("/proc/net/udp")), 1);
    }

    #[test]
    fn not_procnet() {
        assert_eq!(is_procnet(Some("/etc/passwd")), 0);
    }

    #[test]
    fn null_arg() {
        assert_eq!(is_procnet(None), 0);
    }
}